installing/installing_bare_metal/installing-restricted-networks-bare-metal.adoc

```diff
@@ -92,6 +92,8 @@ include::modules/installation-operators-config.adoc[leveloffset=+1]
 
 include::modules/installation-registry-storage-config.adoc[leveloffset=+2]
 
+include::modules/registry-change-management-state.adoc[leveloffset=+3]
```

Comment:
@bmcelvee I know you're only including the file that already exists, but I wasn't sure if it was already peer reviewed or not. So just a few quick thoughts on it in case you care to take a look:

* For the title of the module, suggest Change -> Changing
* The way it's worded seems a little confusing. Is the operator called the "ManagementState Image Registry Operator"? Or are you just changing the managementState field of the Image Registry Operator? If the latter, I'd suggest rewording throughout the module to something a little clearer. For example, "Changing the image registry's management state" for the title, and "To start the image registry, you must change the Image Registry Operator configuration's `managementState` from `Removed` to `Managed`."

Reply:
Thank you! It was peer reviewed a while back, but I don't mind updating it.